README.md

```diff
@@ -173,19 +178,264 @@ The explanation functions accept both models and pipelines as input. If a model
+
+    ```python
+    from interpret.ext.blackbox import TabularExplainer
+
+    # "features" and "classes" fields are optional
+    explainer = TabularExplainer(model, 
+                                 x_train, 
+                                 features=breast_cancer_data.feature_names, 
+                                 classes=classes)
+    ```
+
+    or
+
+    ```python
+
+    from interpret.ext.blackbox import MimicExplainer
+    
+    # you can use one of the following four interpretable models as a global surrogate to the black box model
+    
+    from interpret.ext.glassbox import LGBMExplainableModel
+    from interpret.ext.glassbox import LinearExplainableModel
+    from interpret.ext.glassbox import SGDExplainableModel
+    from interpret.ext.glassbox import DecisionTreeExplainableModel
+
+    # "features" and "classes" fields are optional
+    # augment_data is optional and if true, oversamples the initialization examples to improve surrogate model accuracy to fit original model.  Useful for high-dimensional data where the number of rows is less than the number of columns. 
+    # max_num_of_augmentations is optional and defines max number of times we can increase the input data size.
+    # LGBMExplainableModel can be replaced with LinearExplainableModel, SGDExplainableModel, or DecisionTreeExplainableModel
+    explainer = MimicExplainer(model, 
+                               x_train, 
+                               LGBMExplainableModel, 
+                               augment_data=True, 
+                               max_num_of_augmentations=10, 
+                               features=breast_cancer_data.feature_names, 
+                               classes=classes)
+    ```
+   or
+
+    ```python
+    from interpret.ext.blackbox import PFIExplainer 
+    
+    # "features" and "classes" fields are optional
+    explainer = PFIExplainer(model, 
+                             features=breast_cancer_data.feature_names, 
+                             classes=classes)
+    ```
+## Overal (Global) feature importance values
+
+Get the global feature importance values.
+    
+```python
+
+# you can use the training data or the test data here
+global_explanation = explainer.explain_global(x_train)
+
+# if you used the PFIExplainer in the previous step, use the next line of code instead
+# global_explanation = explainer.explain_global(x_train, true_labels=y_test)
+
+# sorted feature importance values and feature names
+sorted_global_importance_values = global_explanation.get_ranked_global_values()
+sorted_global_importance_names = global_explanation.get_ranked_global_names()
+dict(zip(sorted_global_importance_names, sorted_global_importance_values))
```

Comment:
I think this line is exactly the same as the call to `get_feature_importance_dict()` below